data/transport missions.txt

```diff
@@ -1412,6 +1415,138 @@ mission "Hallucination"
 
 
 
+mission "Shady passenger transport 1"
+	name "Shady passenger transport"
+	description "Discreetly transport a passenger to <destination>. If you are caught with him, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" "core" "urban"
+	destination
+		attributes "frontier" "dirt belt" "rim" "north" "south" "pirate"
+		distance 2 3
+	passengers 1
+	illegal 75000 `Your passenger turns out to be a small-time wanted criminal. As he is hauled off in chains, you vehemently protest that you knew nothing of his shady history. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
+
+	to offer
+		random < 10
+		"rejected illegal jobs" < 3
+
+	on offer
+		dialog `While having a drink at one of one of the spaceport's bars, a somewhat scruffy-looking man in a dark cloak pulls you aside and offers <payment> for discreet, no-questions-asked passage to <destination>, which is quite close by. This could be a quick and lucrative job. (It also sounds illegal, so you'll need to avoid getting scanned or landing on planets with high security.)`
+
+	on accept
+		"rejected illegal jobs" --
+
+	on decline
+		"rejected illegal jobs" ++
+
+	on complete
+		"getaway driver" ++
+		payment 50000
+		dialog `Your mysterious passenger has spoken hardly a word during the entire trip. As soon as you land, he hands you your payment of <payment> and disappears into a crowd, off to do who-knows-what.`
```

Comment:
`off to do who-knows-what` is used here and in shady transport 2. You might want to change one of them to something different.